app/src/main/java/com/example/ithaca_transit_android_v2/RouteOptionsActivity.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.ithaca_transit_android_v2
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+
+class RouteOptionsActivity : AppCompatActivity() {
```

Comment:
does this need to be an activity, or can it exist as a fragment or view delegate?

Reply:
Will try and address this